bin/functions/load_config.py

```diff
@@ -606,11 +610,10 @@ def probe_java_opts():
     cnt = 0
     map_java_opts_line = ""
     reduce_java_opts_line = ""
-    try:
-        with open(file_name) as f:
-            content = f.read()
-    except IOError:
-        return
+    lines = read_file_content(file_name)
+    content = ""
+    for line in lines:
+        content = content + line
```

Comment:
In this case, we don't need to add a '\n', because the code below doesn't deal with '\n' in the middle of content, while it just split according to '<value>'